src/main/java/net/sf/jabref/gui/specialfields/SpecialFieldIcon.java

```diff
@@ -0,0 +1,75 @@
+package net.sf.jabref.gui.specialfields;
+
+import javax.swing.Icon;
+import javax.swing.JLabel;
+
+import net.sf.jabref.gui.IconTheme;
+import net.sf.jabref.logic.specialfields.SpecialField;
+import net.sf.jabref.model.entry.SpecialFieldValue;
+
+
+public class SpecialFieldIcon {
+
+    public static Icon getRepresentingIcon(SpecialField field) {
+        if (SpecialField.PRINTED.equals(field)) {
```

Comment:
I think a switch-case would be more readable and since java 7 it is possible to directly a switch on Strings:
http://docs.oracle.com/javase/7/docs/technotes/guides/language/strings-switch.html
It uses the equals method